doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,17 @@
+= Coordinate systems and reference points
+
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+OSI uses three coordinate systems:
+
+World coordinate system::
+Coordinate system for all quantities which are part of ground truth.
+The world coordinate system is an inertial x/y/z-coordinate system.
+
+Sensor coordinate system::
+Coordinate system for all quantities which are part of sensor data.
+The origin is the mounting position of the physical sensor or a virtual mounting position, depending on the technology.
+
+Object coordinate system::
+Coordinate system for local object coordinates.
```

Comment:
@ThomasNaderBMW 
"Coordinate system for local object coordinates", stating that a "coordinate system" is for "coordinates" is not necessary. Suggestion from my side:
**Local coordinate system for objects (or Assets).**

Reply:
CCB Meeting 04.08.2021:
- "Local object coordinate system"